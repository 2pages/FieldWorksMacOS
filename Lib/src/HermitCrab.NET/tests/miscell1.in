;This is file MISCELL1.IN.  From here, we'll recursively call the file
; MISCELL2.IN.  Before termination, you should see the msg
; "End of file MISCELL1.IN.".
;About to open TESTS\MISCELL2.IN as input; should see msg m6508.
(morpher_input_from_file message c:\data\hc\morpher\tests\MISCELL2.IN)
;Ensure that we can't open for writing a file that is already open:
; (error msg 6031):
(morpher_output_to_file message c:\data\hc\morpher\tests\MISCELL.OUT)
;End of file MISCELL1.IN; should see msg m6521:
