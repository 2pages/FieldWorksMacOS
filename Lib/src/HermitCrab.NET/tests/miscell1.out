;This line should appear as first line in "miscell1.out"
;Ensure that we can't open for reading a file that is open for output
; (error msg 6030):
ERROR:hc6030 --- <hc6030 fname c:\data\hc\morpher\tests\MISCELL.OUT >
;Test loading of msg file.  First try one that does't exist (err msg 6049):
ERROR:hc6049 --- <hc6049 fname c:\data\hc\morpher\nonesuch >
;Now try loading a real file (msg 6514):
((
 <hc6514 fname c:\data\hc\morpher\messages.ari > ) )
;Trigger an error and see if we get the full text of the msg:
ERROR:hc6001 --- Morpher error: Unknown morpher variable: nonesuch.
;Get rid of the msgs (in case we have further tests):
(Prolog command finished )
;Now we'll revert to stdout.  Should get msg m6522:
(message (
 <hc6522 fname standard out > ) )
;This line should also be in "miscell1.out"
;Now we'll close this file for output and revert to stdout; msg m6522:
(message (
 <hc6522 fname standard out > ) )
